### PR TITLE
Add Longhorn 1.5.3 chart with dev-2.8

### PR DESCRIPTION
**Pull request by mantissahz**
release longhorn v1.5.3 into Rancher 2.8

Longhorn longhorn/longhorn#7082
Rancher rancher/rancher#43567

Signed-off-by: James Lu <james.lu@suse.com>

cc @innobead @PhanLe1010 @khushboo-rancher @roger-ryao

### CODE REVIEW

**Comment by mantissahz** [comment]
Self-Verified: PASSED

Install:
Screenshot_20231120_221759

Workload & RecurringJob Backup:
2023-11-20-pm10 25 25

Uninstall:
Screenshot_20231120_222927

**Comment by roger-ryao** [comment]
Prereq: Set Concurrent Automatic Engine Upgrade Per Node Limit to greater than 0
Test steps:
- [x] 1. Fresh installation
- [x] 2. Uninstallation
- [x] 3. Upgrade from v1.4.4
- [x] 4. Upgrade from v1.5.1

File: charts/longhorn/103.2.1+up1.5.3/questions.yaml.orig [hunk 1]
```diff
@@ -0,0 +1,890 @@
+categories:
```
Comment (PhanLe1010) [inline charts/longhorn/103.2.1+up1.5.3/questions.yaml.orig]: This file is unnecessary. Can we make sure to cleanup and return the chart making follow https://confluence.suse.com/display/LON/How+to+create+Rancher+Chart+PRs

Comment (mantissahz) [reply charts/longhorn/103.2.1+up1.5.3/questions.yaml.orig]: Removed files and updated the confluence.
Thanks

**Review by PhanLe1010 — APPROVED** [review]

**Comment by roger-ryao** [comment]
Prereq: Set Concurrent Automatic Engine Upgrade Per Node Limit to greater than 0
verify is passed this PR as follows step
- [x] Fresh installation
- [x] Uninstallation
- [x] Upgrade from v1.5.1 - > v1.5.3
- [x] Upgrade from v1.4.4 - > v1.5.3

**Pull request merged**